src/logic/tokens/store/actions/fetchSafeTokens.ts

```diff
@@ -73,19 +67,19 @@ const fetchSafeTokens = (safeAddress: string) => async (
 ): Promise<void> => {
   try {
     const state = getState()
-    const safe = safeSelector(state)
-    const currentTokens = tokensSelector(state)
+    const safe = state[SAFE_REDUCER_ID].getIn(['safes', safeAddress])
+    const currentTokens = state[TOKEN_REDUCER_ID]
 
     if (!safe) {
       return
     }
 
     const result = await backOff(() => fetchTokenCurrenciesBalances(safeAddress))
-    const currentEthBalance = safeEthBalanceSelector(state)
-    const safeBalances = safeBalancesSelector(state)
-    const alreadyActiveTokens = safeActiveTokensSelector(state)
-    const blacklistedTokens = safeBlacklistedTokensSelector(state)
-    const currencyValues = currencyValuesSelector(state)
+    const currentEthBalance = safe.get('ethBalance')
+    const safeBalances = safe.get('balances')
+    const alreadyActiveTokens = safe.get('activeTokens')
+    const blacklistedTokens = safe.get('blacklistedTokens')
+    const currencyValues = state[CURRENCY_VALUES_KEY]
```

Comment:
why was this changed from selectors to direct assess? so that if we change one property we'll also have to fix it there instead only the selector?